src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -124,6 +131,8 @@
         public static final TrigonTalonSRX MOTOR = TransporterComponents.MOTOR;
         public static final ColorSensorV3 COLOR_SENSOR = TransporterComponents.COLOR_SENSOR;
         public static final int STALL_CURRENT_LIMIT = 20;
+        //TODO: set reasonable power
+        public static final double POWER = 0.5;
```

Comment:
1. climber is not related
2. defualt power not power

Reply:
it already was in transporter